jest.config.js

```diff
@@ -166,14 +166,15 @@ module.exports = {
 
   // A map from regular expressions to paths to transformers
   transform: {
-    '^.+\\.(js|jsx)$': 'babel-jest',
+    '^.+\\.js$': 'babel-jest',
+    '^.+\\.svelte$': 'jest-transform-svelte',
     '.+\\.(css|styl|less|sass|scss)$': 'jest-transform-css'
-  },
+  }
 
   // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
-  transformIgnorePatterns: [
-    'node_modules/?!(lodash-es)/'
-  ]
+  // transformIgnorePatterns: [
```

Comment:
remove

Reply:
The jest config is 99% commented out defaults, so just left this there as a comment